### ISSUE TITLE
tqdm: errors in the tests with "AssertionError: attempt to release recursive lock not owned by thread"

### ISSUE THREAD

**Issue by yarikoptic** [dandi/dandi-cli#111]
Not a good day for tqdm I guess. Now (after #110 on smaug) it errors tests locally 
```
_________________________________________ test_download_multiple_files __________________________________________

tmpdir = local('/home/yoh/.tmp/pytest-of-yoh/pytest-657/test_download_multiple_files0')

    @mark.skipif_no_network
    def test_download_multiple_files(tmpdir):
        url = (
            "https://gui.dandiarchive.org/#/folder/5e70d3173da50caa9adaf334/selected/"
            "item+5e70d3173da50caa9adaf335/item+5e70d3183da50caa9adaf336"
        )
    
>       ret = download(url, tmpdir)

dandi/tests/test_download.py:74: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
dandi/download.py:230: in download
    for file in files
venvs/dev3/lib/python3.7/site-packages/joblib/parallel.py:1017: in __call__
    self.retrieve()
venvs/dev3/lib/python3.7/site-packages/joblib/parallel.py:909: in retrieve
    self._output.extend(job.get(timeout=self.timeout))
/usr/lib/python3.7/multiprocessing/pool.py:657: in get
    raise self._value
/usr/lib/python3.7/multiprocessing/pool.py:121: in worker
    result = (True, func(*args, **kwds))
venvs/dev3/lib/python3.7/site-packages/joblib/_parallel_backends.py:608: in __call__
    return self.func(*args, **kwargs)
venvs/dev3/lib/python3.7/site-packages/joblib/parallel.py:256: in __call__
    for func, args, kwargs in self.items]
venvs/dev3/lib/python3.7/site-packages/joblib/parallel.py:256: in <listcomp>
    for func, args, kwargs in self.items]
dandi/girder.py:389: in download_file
    self.downloadFile(file_id, path)
venvs/dev3/lib/python3.7/site-packages/girder_client/__init__.py:1207: in downloadFile
    length=int(req.headers.get('content-length', 0))) as reporter:
dandi/girder.py:490: in __init__
    self._pbar = tqdm.tqdm(desc=label, total=length, unit="B", unit_scale=True)
venvs/dev3/lib/python3.7/site-packages/tqdm/std.py:1048: in __init__
    self.refresh(lock_args=self.lock_args)
venvs/dev3/lib/python3.7/site-packages/tqdm/std.py:1338: in refresh
    self._lock.release()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <tqdm.std.TqdmDefaultWriteLock object at 0x7fb21a006b10>

    def release(self):
        for lock in self.locks[::-1]:  # Release in inverse order of acquisition
>           lock.release()
E           AssertionError: attempt to release recursive lock not owned by thread

venvs/dev3/lib/python3.7/site-packages/tqdm/std.py:93: AssertionError
--------------------------------------------- Captured stderr call ----------------------------------------------
2020-06-01 18:36:24,494 [    INFO] Downloading item with id ['5e70d3173da50caa9adaf335', '5e70d3183da50caa9adaf336'] from https://girder.dandiarchive.org/
sub-anm372795_ses-20170715.nwb: 100%|██████████| 323k/323k [00:00<00:00, 422kB/s]
```
```
$> acpolicy python3-tqdm
python3-tqdm:
  Installed: 4.43.0-1
  Candidate: 4.43.0-1
  Version table:
 *** 4.43.0-1 900
        900 http://deb.debian.org/debian bullseye/main amd64 Packages
        900 http://deb.debian.org/debian bullseye/main i386 Packages
        600 http://http.debian.net/debian sid/main amd64 Packages
        100 /var/lib/dpkg/status
```

**Comment by yarikoptic** [comment]
seems to be some intermittent (race?) issue -- passed on rerun.  I will close for now - but now we have a record at least. dandi-cli is at 0.4.6-11-gda610ba

**Comment by yarikoptic** [comment]
happened also on a recent CI run on OSX: https://github.com/dandi/dandi-cli/runs/729050251 
I wonder if anything recently "introduced" by tqdm. Current master of tqdm is v4.46.0-2-g571f0ed

**Comment by yarikoptic** [comment]
FWIW, fixed in tqdm yesterday in ` v4.50.1-2-g572a8e8` AKA `v4.50.2^2~7`
